plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -680,6 +689,7 @@ class NewCreateServiceModalForm extends Component {
                         onConvertToPod={onConvertToPod}
                         service={service}
                         onRemoveItem={this.handleRemoveItem}
+                        onClickContainer={this.handleClickContainer}
```

Comment:
Let's keep it consistent with the other names `onRemoveItem` `onAddItem ` so I would name `onClickItem` and `handleClickItem`